tools/mo/openvino/tools/mo/back/ie_ir_ver_2/emitter.py

```diff
@@ -513,7 +513,7 @@ def serialize_network(graph, net_element, unsupported):
             continue
 
         # Here output data node count is checked. Each port cannot have more than one data node.
-        assert len(node.out_nodes()) == 1, "Incorrect graph. Non-Result node with name {} " \
+        assert len(node.out_nodes()) >= 1, "Incorrect graph. Non-Result node with name {} " \
```

Comment:
this is questionable fix because it was early working for different operations like `MaxPool`, `TopK`. Do we really root-cause the problem or did anybody change this code very recently?

Reply:
We know the root cause - in 2021.4 there was not such assert (assert len(node.out_nodes()) == 1) it was introduced in 2022.1 and caused bugs for ops with 2 and more outputs. It's totally ok if at this moment op contain 2 outputs (either real one or faked). After emitter.py the fake Result with data node are removed and we get the correct IR.

![image](https://user-images.githubusercontent.com/5703039/209318374-ca30792b-ff01-4798-8e86-941c25d4728b.png)